packages/perseus/src/widgets/graded-group/graded-group.tsx

```diff
@@ -170,9 +186,16 @@ export class GradedGroup
 
     _checkAnswer: () => void = () => {
         this.rendererRef.current?.showRationalesForCurrentlySelectedChoices();
-        const score: PerseusScore = this.rendererRef.current?.score() || {
-            type: "invalid",
-        };
+        // const score: PerseusScore = this.rendererRef.current?.score() || {
+        //     type: "invalid",
+        // };
+
+        const userInput = this.rendererRef.current?.getUserInputMap();
+        const score: PerseusScore = this.props.gradingCallback(
+            this.props.widgetId,
```

Comment:
In thinking about this a bit more, I'm not sure that'll work. The props may not have the data to score. I do think it'll be slightly more complicated than just an id though. These widgets can show up inside hints, or the `graded-group-set` or articles. We'll need a way to uniquely identify where this widget lives so that the chrome can correctly identify which scoring data to use.